massgov/mayflower
# React/DP-14340:  emergency alert molecule.

## Description
This PR addresses the following:
- (React) [EmergencyAlert] DP-14340: Adds the EmergencyAlert molecule from pattern lab.
- (React) [Link] DP-14340: Adds support for using props.children with Link.
- (React) [Link] DP-14340: Adds support for passing custom classes to Link component.
- (React) [Link] DP-14340: Sets default text to empty string.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-14340)

## Steps to Test


1. Pull the branch. Browse to molecules -> EmergencyAlert and verify that the component works as intended.

## Screenshots
<img width="1472" alt="Screen Shot 2019-06-11 at 4 06 51 PM" src="https://user-images.githubusercontent.com/5257660/59302994-4643a500-8c63-11e9-85f3-07d6a2baf0a0.png">
